images.v2.yaml

```diff
@@ -1,328 +1,76 @@
-tensorflow/tensorflow:1.5.0-devel-gpu-py3:
-  isRecommended: False
-  description: 'TensorFlow 1.5.0 developer version on Python 3 with GPU support'
-  tags:
-    - 'python 3'
-    - 'tensorflow 1.5.0'
-    - 'GPU'
-    - 'developer version'
-tensorflow/tensorflow:1.9.0-devel-gpu-py3:
-  isRecommended: False
-  description: 'TensorFlow 1.9.0 developer version on Python 3 with GPU support'
-  tags:
-    - 'python 3'
-    - 'tensorflow 1.9.0'
-    - 'GPU'
-    - 'developer version'
-tensorflow/tensorflow:1.9.0-devel-py3:
-  isRecommended: False
-  description: 'TensorFlow 1.9.0 developer version on Python 3 without GPU support'
-  tags:
-    - 'python 3'
-    - 'tensorflow 1.5.0'
-    - 'CPU'
-    - 'developer version'
-tensorflow/tensorflow:1.13.1-gpu-py3:
-  isRecommended: False
-  description: 'TensorFlow 1.13.1 on Python 3 with GPU support'
-  tags:
-    - 'python 3'
-    - 'tensorflow 1.13.1'
-    - 'GPU'
+pytorch/pytorch:1.10.0-cuda11.3-cudnn8-runtime:
+  description: PyTorch 1.10.0 on CUDA 11.3
+  isRecommended: true
+  tags:
+  - PyTorch
+  - PyTorch 1.10.0
+  - GPU
```

Comment:
Should we add a CUDA tag? as now there is a small mismatch with description and tags, where one separates between more broadly GPU support and the other is more specifically about CUDA support.

Reply:
We don't really use the tags for anything, so maybe if we do, then...